_episodes/15-transferring-files.md

```diff
@@ -172,7 +172,9 @@ To connect to the cluster, we'll just need to enter our credentials at the top o
 
 Hit "Quickconnect" to connect. You should see your remote files appear on the right hand side of the
 screen. You can drag-and-drop files between the left (local) and right (remote) sides of the screen
-to transfer files.
+to transfer files. 
+
+Finally, if you need to move large files (typically larger than a gigabyte) from one remote computer to another remote computer, using `scp` or `rsync` between the two as this will likely be more efficient than using Filezilla (or related services).
```

Comment:
```suggestion
Finally, if you need to move large files (typically larger than a gigabyte) from one remote computer to another remote computer, SSH in to the computer hosting the files and use `scp` or `rsync` to transfer over to the other. This will be more efficient than using FileZilla (or related applications) that would copy from the source to your local machine, then to the destination machine.
```

Reply:
Thanks.